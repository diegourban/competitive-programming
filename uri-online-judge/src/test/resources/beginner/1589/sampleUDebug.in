15
1 2
2 3
3 4
4 5
5 6
6 7
9 8
3 3
4 5
12 32
999 999
321 321
215 635
324 909
421 242