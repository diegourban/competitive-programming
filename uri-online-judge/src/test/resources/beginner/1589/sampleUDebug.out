3
5
7
9
11
13
17
6
9
44
1998
642
850
1233
663
